src/components/App/index.tsx

```diff
@@ -17,32 +17,40 @@ const App = props => {
 
   useEffect(() => {
     httpGet(
-      constants.API_HEALTHCHECK_HELLO,
+      constants.API_MIRROR_HEALTHCHECK_HELLO,
```

Comment:
You should make both calls in parallel to avoid performance delay. Have four separate variables to capture the outcome. 
oneauthVerified
mirrorVerified
oneauthReachable
mirrorReachable